### ISSUE TITLE
Dtype test failure of Google Options.

### ISSUE THREAD

**Issue by sinhrks** [pydata/pandas-datareader#210]
Recent master build fails:

```
======================================================================
FAIL: test_get_options_data (pandas_datareader.tests.test_google_options.TestGoogleOptions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/pydata/pandas-datareader/pandas_datareader/tests/test_google_options.py", line 43, in test_get_options_data
    self.assert_option_result(options)
  File "/home/travis/build/pydata/pandas-datareader/pandas_datareader/tests/test_google_options.py", line 34, in assert_option_result
    tm.assert_series_equal(df.dtypes, pd.Series(dtypes, index=exp_columns))
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/pandas/util/testing.py", line 681, in assert_series_equal
    assert_almost_equal(left.values, right.values, check_less_precise)
  File "pandas/src/testing.pyx", line 58, in pandas._testing.assert_almost_equal (pandas/src/testing.c:2906)
  File "pandas/src/testing.pyx", line 93, in pandas._testing.assert_almost_equal (pandas/src/testing.c:1917)
  File "pandas/src/testing.pyx", line 143, in pandas._testing.assert_almost_equal (pandas/src/testing.c:2779)
AssertionError: dtype('float64') != dtype('int64')
```

**Comment by femtotrader** [comment]
A similar issue

```
$ nosetests -s -v pandas_datareader/tests/test_google_options.py
test_expiry_dates (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_all_data (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_call_data (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_forward_data (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_near_stock_price (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_options_data (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... FAIL
test_get_options_data_with_year (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_options_data_yearmonth (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_put_data (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok

======================================================================
FAIL: test_get_options_data (pandas_datareader.tests.test_google_options.TestGoogleOptions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/femto/github/femto/pandas-datareader/pandas_datareader/tests/test_google_options.py", line 43, in test_get_options_data
    self.assert_option_result(options)
  File "/Users/femto/github/femto/pandas-datareader/pandas_datareader/tests/test_google_options.py", line 34, in assert_option_result
    tm.assert_series_equal(df.dtypes, pd.Series(dtypes, index=exp_columns))
  File "/Users/femto/anaconda/lib/python3.5/site-packages/pandas/util/testing.py", line 1049, in assert_series_equal
    check_less_precise, obj='{0}'.format(obj))
  File "pandas/src/testing.pyx", line 58, in pandas._testing.assert_almost_equal (pandas/src/testing.c:3887)
  File "pandas/src/testing.pyx", line 147, in pandas._testing.assert_almost_equal (pandas/src/testing.c:2769)
  File "/Users/femto/anaconda/lib/python3.5/site-packages/pandas/util/testing.py", line 915, in raise_assert_detail
    raise AssertionError(msg)
AssertionError: Series are different

Series values are different (10.0 %)
[left]:  [float64, float64, float64, float64, float64, float64, float64, object, float64, datetime64[ns]]
[right]: [float64, float64, float64, float64, float64, float64, int64, object, float64, datetime64[ns]]
-------------------- >> begin captured logging << --------------------
requests.packages.urllib3.connectionpool: DEBUG: "GET /finance/option_chain?q=GOOG&expd=16&expm=9&expy=2016&output=json HTTP/1.1" 200 None
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 9 tests in 5.217s

FAILED (failures=1)
✘-1 ~/github/femto/pandas-datareader [fix_test_google_options|✚ 1…7]
21:50 $ nosetests -s -v pandas_datareader/tests/test_google_options.py
test_expiry_dates (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_all_data (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_call_data (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_forward_data (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_near_stock_price (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_options_data (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_options_data_with_year (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_options_data_yearmonth (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok
test_get_put_data (pandas_datareader.tests.test_google_options.TestGoogleOptions) ... ok

----------------------------------------------------------------------
Ran 9 tests in 0.202s

OK
```

A first run catch it not a second run! that's odd!

**Comment by davidastephens** [comment]
Fixed in #270
